ui/swiss/css/_player-info.scss

```diff
@@ -129,17 +129,17 @@
       max-width: 200px;
     }
 
-    td:last-child {
+    td:result {
```

Comment:
I never heard of the `:result` pseudo-selector before. Are you sure?

Reply:
There’s a result class used here, maybe that?

Reply:
Yeah, it is that. It had previously used last-child just to colour the result, as far as I can tell. But the berserk icon for the player is the last element now, so I added a result class to handle the styling for the result.

Old: https://github.com/lichess-org/lila/blob/21849edf64dd4c9bbdf0ab3c6ede37539fb89cdc/ui/tournament/css/_player-info.scss#L49-L62 

New: https://github.com/jenningsfan/lila/blob/5b27ce0610c85b329fcccf4df5e18a61e23682d1/ui/tournament/css/_player-info.scss#L49-L62

```scss
    td.result {
      font-weight: bold;
      opacity: 0.8;
    }


    tr.win td.result {
      color: $c-good;
      opacity: 1;
    }


    tr.loss td.result {
      color: $c-bad;
      opacity: 1;
    }
```

Reply:
Oh, I just realised that I'd used a colon there instead of a full stop. 🤦. Will fix that now

Reply:
Fixed